### PR TITLE
Created UI for listing note

### LINKED ISSUE #1: Create note listing page UI

### PR DESCRIPTION
**Pull request by kamolesh97**
closes #1 

@yedhink _a please review.

https://user-images.githubusercontent.com/71279916/217298032-1ee4d1d3-1081-479c-9c3e-c83f34fad521.mov

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/MenuBar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const Menu = ({ showMenu }) => {
+  const { t } = useTranslation();
+
+  return (
+    <MenuBar showMenu={showMenu} title={t("common.notes")}>
+      {VIEWS.map(({ category, count }) => (
+        <MenuBar.Block
+          active={category === VIEWS[0].category}
+          count={count}
+          key={category}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/MenuBar/index.jsx:17]: Is `category` a primitive value or a compound value? Because we are supposed to pass a primitive value like say a Number or String to `key` prop. Please read more upon the necessity of the `key` prop and how it's utilised by React. Refer: https://academy.bigbinary.com/learn-react/react-best-practices/using-unique-keys-for-list-elements

The reason why I even had to ask the above question was because of the non-semantic naming convention that you have followed here. The variable name `category` suggests in all aspects that it's an `Object`. If it was the name of the category, then it ought to have been named `categoryName`. Or if it was meant to be an object itself then pass `category.name` to `key` prop.

File: app/javascript/src/translations/fr.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "greeting": "France"
+}
```
Comment (yedhink) [inline app/javascript/src/translations/fr.json:3]: I don't understand the point of this translation file. Is this file needed?

**Review by yedhink — COMMENTED** [review]
@kamolesh97

File: app/javascript/src/translations/fr.json (hunk 2 above)
Comment (kamolesh97) [reply app/javascript/src/translations/fr.json:3]: Nope, this file is not needed, just that I copied the translations dir from another repo which had this file, will remove it.

File: app/javascript/src/components/Dashboard/Notes/MenuBar/index.jsx (hunk 1 above)
Comment (kamolesh97) [reply app/javascript/src/components/Dashboard/Notes/MenuBar/index.jsx:17]: Got it, my bad, will fix this convention issue.
